project_access_tokens.go

```diff
@@ -88,9 +89,10 @@ func (s *ProjectAccessTokensService) ListProjectAccessTokens(pid interface{}, op
 // GitLab API docs:
 // https://docs.gitlab.com/ee/api/resource_access_tokens.html#create-a-project-access-token
 type CreateProjectAccessTokenOptions struct {
-	Name      *string  `url:"name,omitempty" json:"name,omitempty"`
-	Scopes    []string `url:"scopes,omitempty" json:"scopes,omitempty"`
-	ExpiresAt *ISOTime `url:"expires_at,omitempty" json:"expires_at,omitempty"`
+	Name        *string          `url:"name,omitempty" json:"name,omitempty"`
+	Scopes      []string         `url:"scopes,omitempty" json:"scopes,omitempty"`
+	AccessLevel AccessLevelValue `url:"access_level,omitempty" json:"access_level,omitempty"`
```

Comment:
This needs to be a pointer to keep everything consistent:

```suggestion
	AccessLevel *AccessLevelValue `url:"access_level,omitempty" json:"access_level,omitempty"`
```